code_generator/End2End/GeneratedDependencies/Generated/improbable/testschema/TypeName.cs

```diff
@@ -94,6 +95,9 @@ public enum NestedEnum : uint
                 }
                 
             }
+            
```

Comment:
where did all these new lines come from?

Reply:
This doesn't seem to be caused by my PR. cc @jamiebrynes7

Reply:
🤷‍♂️